int64 id
int64 x
int64 y
int64 height
int64 width
int64 active